packages/world/ts/concatBaseAbi.ts

```diff
@@ -0,0 +1,16 @@
+import { Abi, AbiItem, AbiFunction, toFunctionSelector } from "viem";
+import IBaseWorldAbi from "../out/IBaseWorld.sol/IBaseWorld.abi.json";
+
+function isAbiFunction(abiItem: AbiItem): abiItem is AbiFunction {
+  return abiItem.type === "function";
+}
+
+export function concatBaseAbi(worldAbi: Abi): Abi {
```

Comment:
sorry to keep going back and forth on this but now that this is only handling functions, I think this logic is worth inlining into `getWorldAbi` rather than being a separate helper

means we'll likely need to mock `getFunctions` if we want to be able to test it, but I think the logic is simple enough to not necessarily need it

Reply:
All good, just removed `concatbaseAbi` and adjusted the tests, lmk what you think